database/insertData/03_template_A_featureShowcase.js

```diff
@@ -1102,6 +1102,161 @@ exports.queries = [
                         default: "cards"
                       }
                     }
+                    {
+                      code: "PB14"
+                      index: 6
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                    }
+                    {
+                      code: "searchUsers"
+                      index: 7
+                      title: "Search Demo (users)"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      helpText: "This page contain examples of the **Search** element. It can make API queries in response to user input and display the results, which can be selected."
+                      parameters: {
+                        label: "Search for users"
+                        description: "Start typing to search database for usernames"
+                        placeholder: "Search usernames"
+                        icon: "user"
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query GetUsers($user: String!) { users(filter: { username: {includesInsensitive: $user } }) {nodes { username, firstName, lastName }}}",
+                            "",
```

Comment:
I see, this is using the local server, should be better to send that `graphqlEndpoint` string instead? Just to it's easier to remember what this field is...

Reply:
Right, you meant the more specific default value (I was confused at first). Anyway, done dcf2e3f